internal/volume/volumes_information.go

```diff
@@ -393,6 +398,10 @@ func (v *VolumesInformation) generateVolumeInfoForCSIVolumeSnapshot() {
 				},
 			}
 
+			if volumeSnapshot.Status.CreationTime != nil {
```

Comment:
`CompletionTimestamp` is missing for CSI volume snapshot

Reply:
CompletionTimestamp is filled when VolumeInfos is updated for async operation.